src/devices/Ahtxx/README.md

```diff
@@ -0,0 +1,16 @@
+﻿# Ahtxx
+
+## Summary
+AHTxx temperature and humidity sensor family.
+
+## Device Family
+The binding supports the following types:
+* AHT10 - http://www.aosong.com/en/products-40.html (not tested, yet)
+* AHT20 - http://www.aosong.com/en/products-32.html
+
+## Binding Notes
+
+The AHT-devices can be accessed as an I2C bus device. However, you can use only one device per bus as the address is fix.
+
+
```

Comment:
Maybe a bit of documentation on how to use the binding? I know there is the sample. Still, maybe something quick can be good.

Reply:
@Ellerbach See my other comment above.